src/pytypes/onefuzztypes/requests.py

```diff
@@ -41,6 +41,11 @@ class JobSearch(BaseRequest):
 
 class NotificationCreate(BaseRequest, NotificationConfig):
     container: Container
+    replace_existing: bool = Field(default=True)
```

Comment:
I suggest also setting this default to `False`

Reply:
```suggestion
    replace_existing: bool = Field(default=False)
```